src/Lint.jl

```diff
@@ -350,7 +356,54 @@ function lintdir{T<:AbstractString}(dir::T, ctx::LintContext=LintContext())
     ctx.messages
 end
 
-function readandwritethestream(conn)
+function convertmsgtojson(msgs, style)
+    if style == "LintMessage"
+        return JSON.json(msgs)
+    end
+    output = []
+    for msg in msgs
+        evar = msg.variable
+        txt = msg.message
+        file = msg.file
+        linenumber = msg.line
+        errorrange = Array[[linenumber, 0], [linenumber, 80]]
```

Comment:
We don't support 0.4 any more so this should be `[[linenumber, 0], [linenumber, 80]]`, i.e. leave out the `Array`.